components/breadcrumbs/w-breadcrumbs.vue

```diff
@@ -13,10 +13,19 @@
 import { h, Fragment } from 'vue'
 import { interleave } from '@warp-ds/core/breadcrumbs'
 import { breadcrumbs as ccBreadcrumbs } from "@warp-ds/css/component-classes"
+import { i18n } from '@lingui/core';
+import { activateI18n } from '../util/i18n';
+import { messages as enMessages} from './locales/en/messages.mjs';
+import { messages as nbMessages} from './locales/nb/messages.mjs';
+import { messages as fiMessages} from './locales/fi/messages.mjs';
+activateI18n(enMessages, nbMessages, fiMessages);
 
-const props = defineProps({
-  ariaLabel: { type: String, default: 'Her er du' }
-});
+const ariaLabel = i18n._(
+  {
+    id: 'breadcrumbs.ariaLabel',
+    message: 'You are here',
```

Comment:
Just curious, if the default language is `en`, and this message is the same as in the `en/messages.po` file, would the engine pick up that string if you only send the id to this method?

Reply:
Yes, because I added `  sourceLocale: 'en',` in the lingui config, I actually didn't need to add text in the `en/messages.po` file. I've tested that whatever we have in the po/mjs files overrides the default text, and then I just added an English text in all the files 😅

Reply:
I'm a bit confused 😅 Does it mean that we could skip this `message` key in the options passed to `i18n`'s method?

Reply:
Hm. But we do have .po files for `en`, and they have `msgstr "some text"` that is the same as the one we pass to the `i18n`'s method in each component. Is it required that we pass `id` and `message` as options to that method? I'm simply wondering why we need to define that English message twice - in the options argument and in the `en/messages.po` file 😄